src/URLS.tsx

```diff
@@ -15,6 +15,7 @@ export default {
   jobposts: '/karriere/',
   profile: '/profil/',
   login: '/logg-inn/',
+  groups: '/grupper',
```

Comment:
```suggestion
  groups: '/grupper/',
```

Alltid / på slutten av url'er ;)